sound/soc/sof/control.c

```diff
@@ -112,6 +139,9 @@ int snd_sof_switch_put(struct snd_kcontrol *kcontrol,
 		cdata->chanv[i].channel = i;
 	}
 
+	if (scontrol->led_ctl.use_leds)
```

Comment:
nit-pick: can we use more than one LED? It doesn't look possible, so is it rather use_led (singular)?

Reply:
should it even be use_mute_led and SOF_TKN_MUTE_LED_USE   SOF_TKN_MUTE_LED_DIRECTION

Reply:
ok, good suggestions

Reply:
done